services/ui-src/src/components/reports/DynamicModalOverlayReportPage.tsx

```diff
@@ -63,7 +63,9 @@ export const DynamicModalOverlayReportPage = ({
   const showAlert =
     report && errorMessage ? getWPAlertStatus(report, entityType) : false;
 
-  const dashTitle = `${verbiage.dashboardTitle}`;
+  const dashTitle = verbiage.dashboardTitle
```

Comment:
will this error if `dashboardTitle` is not defined? do we need an optional chain?
```suggestion
  const dashTitle = verbiage?.dashboardTitle
```

Reply:
Is the ternary needed? Does `dashTitle` print out if it's undefined?

Reply:
Yeah, that's what I was seeing...I removed the `dashboardTitle` property for one area of the SAR and was seeing "undefined" as the heading in the UI. Trying to find a good way around that.

Reply:
ok it looks like if I take out the interpolation and add the optional chaining, I do not need the ternary. Will see about pushing up this fix!